astro/src/content/quickstarts/quickstart-javascript-nuxt-web.mdx

```diff
@@ -0,0 +1,198 @@
+
+1. Homepage
+1. Account - protected
+1. Makechange - protected
+
+Create a new application using the `npx`.
+
+```shell
+npx nuxi@latest init changebank
+```
+
+Make sure you are in your new directory `changebank`.
+
+```shell
+cd changebank
+```
+
+Install [@sidebase/nuxt-auth](https://sidebase.io/nuxt-auth/v0.6/getting-started/installation), which simplifies integrating with FusionAuth and creating a secure web application.
+
+```shell
+npm install next-auth
+```
+
+Copy environment variables from our complete application example.
+
+```shell
+cp ../complete-application/.env.example .env.local
+```
+
+Also copy an image file into a new directory within `public` called `img`.
+
+```shell
+mkdir ./public/img && cp ../complete-application/public/img/money.jpg ./public/img/money.jpg
+```
+
+## Authentication
+
+Update `nuxt.config.ts` to include the new `@sidebase/nuxt-auth` and include the provider type of `authjs` just like below. 
+
+<Aside type="note">
+If you would like to code with any of the pages prior to enabling authentication, just set `globalAppMiddleware` to false.
+</Aside>
+
+<RemoteCode url={frontmatter.codeRoot + "/complete-application/nuxt.config.ts"}
+  lang="ts" />
+
+Nuxt 3 includes <a href="https://nuxt.com/docs/guide/directory-structure/server#server-routes" target="_blank">Route Handlers</a>, which are the preferred way to handle REST-like requests. In the `Changebank` application you can configure NextAuth.js FusionAuth's provider in a new route handler by creating a file within `/server/api/auth/[...].ts`.
+
+On first load of Nuxt this file will make sure that you have all of the correct environment variables. The variables are then exported in an object called `authOptions` which can be imported on the server when you need to get our session using `getServerSession`.
+
+The `FusionAuthProvider` is then provided to `NextAuth` as a provider for any `GET` or `POST` commands that are sent to the `/api/auth/*` route.
+
+Create a new file named `/server/api/auth/[...].ts` and copy the following code for the ChangeBank application.
+
+<RemoteCode url={frontmatter.codeRoot + "/complete-application/server/api/auth/[...].ts"}
+  lang="ts" />
+
+## App Customization
+
+### Styles
+
+Create a new file named `/changebank/assets/css/global.css` and copy the below CSS for the ChangeBank application.
+
+<RemoteCode url={frontmatter.codeRoot + "/complete-application/assets/css/global.css"}
+  lang="css" />
+
+The above styles will be imported into a new layout that you will create called `/changebank/layouts/default.vue`. Update this file to match the code below. This will allow the rest of the pages within our application and the layout itself to use the global styles. This is the first time that you will use the [useAuth](https://sidebase.io/nuxt-auth/v0.6/application-side/session-access-and-management) Composable from Sidebase. The status can then be used to determine what links should be shown in the layout.
```

Comment:
```suggestion
The above styles will be imported into a new layout that you will create called `/changebank/layouts/default.vue`. Update this file to match the code below. This will allow the rest of the pages within your application and the layout itself to use the global styles. This is the first time that you will use the [useAuth](https://sidebase.io/nuxt-auth/v0.6/application-side/session-access-and-management) Composable from Sidebase. The status can then be used to determine what links should be shown in the layout.
```